.circleci/continue_config.yml

```
@@ -77,8 +77,78 @@ jobs:
             ~/Qt/Tools/CMake/CMake.app/Contents/bin/cmake --build . --target package
             mkdir upload
             cp gpt4all-installer-* upload
+      # persist the unsigned installer
       - store_artifacts:
           path: build/upload
+      # add workspace so signing jobs can connect & obtain dmg
+      - persist_to_workspace:
+          root: build
+          # specify path to only include components we want to persist
+          # accross builds
+          paths:
+            - upload
+
+  sign-offline-chat-installer-macos:
+    macos:
+      xcode: 14.0.0
+    steps:
+      - checkout
+      # attach to a workspace containing unsigned dmg
+      - attach_workspace:
+          at: build
+      - run:
+          name: "Setup Keychain"
+          command: |
+            echo \<<< $MAC_SIGNING_CERT > certs.pem
+            openssl pkcs12 -export -out cert.p12 -in certs.pem -passin $MAC_SIGNING_CERT_PWD
```

These variable references should be wrapped in double quotes, or special characters will end up unescaped:
```suggestion
            echo \<<< "$MAC_SIGNING_CERT" > certs.pem
            openssl pkcs12 -export -out cert.p12 -in certs.pem -passin "$MAC_SIGNING_CERT_PWD"
```